promsonnet/README.md

```diff
@@ -0,0 +1,65 @@
+#PromSonnet
+
+`PromSonnet` is intended as a very simple library for creating Prometheus
+alerts and rules. It is 'patching friendly', as in, it maintains the
+rules internally as a map, which allows users to easily patch alerts and
+recording rules after the fact. (With lists, this requires iteration, and
+is complex.
+
+Take this example:
+
+```
+local prom = import 'prom.libsonnet';
+local promRuleGroupSet = prom.v1.ruleGroupSet;
+local promRuleGroup = prom.v1.ruleGroup;
+{
+  prometheus_up::
+    promRuleGroup.new('prometheus_up')
+    + promRuleGroup.rule.new(
+      'PrometheusUp', {
+        alert: 'PrometheusUp',
+        expr: 'up',
```

Comment:
This should be something like `up{job="prometheus"}==0`, as already changed in example.jsonnet.